packages/helm-charts/blockscout/templates/blockscout-indexer.deployment.yaml

```diff
@@ -86,12 +89,15 @@ spec:
         - name: DISABLE_WEBAPP
           value: "true"
         - name: POOL_SIZE
-          value: {{ .Values.blockscout.indexer.pool_size | quote }}
+          value: {{ .Values.blockscout.indexer.poolSize | quote }}
         - name: METRICS_ENABLED
           value: "{{.Values.blockscout.metrics.enabled}}"
-{{ include "celo.blockscout-env-vars" . | indent 8 }}
-{{- $data := dict "Values" .Values "Database" .Values.blockscout.indexer.db }}
-{{ include "celo.blockscout-db-sidecar" $data | indent 6 }}
+{{- $data := dict "Release" .Release "Values" .Values "Database" .Values.blockscout.indexer.db }}
+{{ include "celo.blockscout.env-vars" $data | indent 8 }}
```

Comment:
this is perhaps outwith the scope of this PR, but I feel we should at some point enforce a convention for helm template var style - older / legacy values seem to have a hyphenated `env-vars` style, whilst our most recent changes are `poolSize` camel case.

Reply:
In this case `env-vars` is the name of a defined template. Template filenames [use dashes](https://v2.helm.sh/docs/chart_best_practices/#templates) according to best practices as well as [chart names](https://helm.sh/docs/chart_best_practices/conventions), so I thought defined templates should use them too.